sections/04_study.md

```diff
@@ -52,7 +52,113 @@ particularly notable in this area?*
 
 ### Protein secondary and tertiary structure
 
-*Jinbo Xu is writing this*
+Proteins play fundamental roles in all biological processes including the 
```

Comment:
I agree with the purpose of this section's introductory paragraph being to first discuss **what** is being predicted, but I think we could be much less verbose. I think all we need here is:

1. One sentence, like you have already, describing what a protein does.
2. Quickly introduce the problem: There are millions of protein sequences, many of which are redundant and few structures are solved.
3. Computational methods as one solution: Over several decades, these algorithms have predicted different aspects of secondary and tertiary structure such as...(as they are listed below) (is there a separate review paper we could cite that discusses protein structure algorithms specifically?)